packages/@okta/vuepress-site/guides/sign-into-web-app/redirect-to-sign-in/aspnetcore/login-redirect.md

```diff
@@ -0,0 +1,54 @@
+When accessing protected resources, ASP.NET Core redirects the user to an Okta sign-in page automatically. You can control which routes are protected with the `[Authorize]` attribute, which is covered in [Require Authentication](../-/require-authentication/).
+
+You can also give the user a **Sign In** button or link. Open your `_Layout.cshtml` file and add code the following code:
```

Comment:
You can also give the user a **Sign In** button or link. Open your `_Layout.cshtml` file and add the following code: